### PR TITLE
Release 1.5.0

### PR DESCRIPTION
**Pull request by minevala**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,46 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+## [1.5.0] - Sep, 22, 2021
+
+### Core
+- [TextInput] Added Info message to inform user of dynamic input value changes
+- [Icons] New icons:
+  - Shopping cart
+
+### React Components
+
+#### Added
```
Comment (VilleMiekkoja) [inline CHANGELOG.md:18]: Should we add the form validation additions to react components?

Comment (minevala) [reply CHANGELOG.md:18]: ?

Comment (minevala) [reply CHANGELOG.md:18]: Now I get it: 
- [ ] Fix react changelog

**Review by ronijaakkola — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,47 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- File Input
+- [Text Input, TextArea, Password Input, Number Input, Time Input, Date Input, File Input] Added Info message to inform user of dynamic input value changes
+- [Icons] New icons:
+  - Shopping cart
+
+#### Change
+- [Accordion] Changed Accordion to open via header click
+- [Accordion] Added support for accordion being open by default
+- Use DateInput and Phone Input in the form examples
```
Comment (VilleMiekkoja) [inline CHANGELOG.md:27]: Phone Input is also validated, though not very strictly. It only requires the international mobile number format. Maybe no need to mention that separately

**Review by laurakarhu — APPROVED** [review]
Yay!

**Review by VilleMiekkoja — APPROVED** [review]
Good job! 👍

File: CHANGELOG.md (hunk 2 above)
Comment (minevala) [reply CHANGELOG.md]: - [ ] add phone input validation text

**Pull request merged**